manifest/owo.Nytra/HeadlessToolPermissionKickCrashFix/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "HeadlessToolPermissionKickCrashFix",
+	"id": "owo.Nytra.HeadlessToolPermissionKickCrashFix",
+	"description": "A Headless Server mod that prevents tool permission kicks from causing a world crash.",
+	"category": "Bug Workaround",
+	"sourceLocation": "https://github.com/Nytra/ResoniteHeadlessToolPermissionKickCrashFix",
+	"versions": {
```

Comment:
I haven't finished up documentation on all the optional mod info sections yet (and some may be subject to change still) but there is "platforms" where you can specify intended platforms. If none are specified, it would be assumed that all are probably fine (within reason of many mods wouldn't make sense for a headless)
https://github.com/resonite-modding-group/resonite-mod-manifest/blob/main/schemas/mod-schema.json#L42C1-L49C5